idb/postgres/postgres.go

```diff
@@ -2396,6 +2402,39 @@ func (db *IndexerDb) Health() (health idb.Health, err error) {
 	}, err
 }
 
+// GetSpecialAccounts is part of idb.IndexerDB
+func (db *IndexerDb) GetSpecialAccounts() (accounts idb.SpecialAccounts, err error) {
+	var cache string
```

Comment:
I'd be tempted to not cache this in metastate but cache it in RAM in the IndexerDb object.

Reply:
I'm actually doing both, it's cached in ram in rewind.go, so GetSpecialAccounts() should only be called once.

I wanted to implement that outside of IndexerDb so that alternative implementations wouldn't need to worry about that optimization.